Userland/Libraries/LibWeb/DOM/Document.cpp

```diff
@@ -628,6 +630,60 @@ NonnullRefPtr<Range> Document::create_range()
     return Range::create(*this);
 }
 
+// https://dom.spec.whatwg.org/#dom-document-createevent
+NonnullRefPtr<Event> Document::create_event(const String& interface)
+{
+    RefPtr<Event> event;
+    if (interface.matches("beforeunloadevent")) {
+        event = Event::create(""); // FIXME: Create BeforeUnloadEvent
+    } else if (interface.matches("compositionevent")) {
+        event = Event::create(""); // FIXME: Create CompositionEvent
+    } else if (interface.matches("customevent")) {
+        event = Event::create(""); // FIXME: Create CustomEvent
+    } else if (interface.matches("devicemotionevent")) {
+        event = Event::create(""); // FIXME: Create DeviceMotionEvent
+    } else if (interface.matches("deviceorientationevent")) {
+        event = Event::create(""); // FIXME: Create DeviceOrientationEvent
+    } else if (interface.matches("dragevent")) {
+        event = Event::create(""); // FIXME: Create DragEvent
+    } else if (interface.matches("event") || interface.matches("events")) {
+        event = Event::create("");
+    } else if (interface.matches("focusevent")) {
+        event = Event::create(""); // FIXME: Create FocusEvent
+    } else if (interface.matches("hashchangeevent")) {
+        event = Event::create(""); // FIXME: Create HashChangeEvent
+    } else if (interface.matches("htmlevents")) {
+        event = Event::create("");
+    } else if (interface.matches("keyboardevent")) {
+        event = Event::create(""); // FIXME: Create KeyboardEvent
+    } else if (interface.matches("messageevent")) {
+        event = Event::create(""); // FIXME: Create MessageEvent
+    } else if (interface.matches("mouseevent") || interface.matches("mouseevents")) {
```

Comment:
As a shorthand for this, we can do:
```suggestion
    } else if (interface.is_one_of("mouseevent", "mouseevents")) {
```